README.md

```diff
@@ -2,26 +2,96 @@
 
 This repository is currently Work In Progress but will eventually be used to manage KubeVirt Enhancement Proposals (VEPs), emphasizing centralized prioritization and enhanced SIG involvement and collaboration.
 
-## Process (DRAFT)
+**WHY**
+The process aims to focus the community's efforts on prioritized pull requests, increase the review bandwidth, and ensure clear visibility of feature progress and associated issues. The relativly new split into SIGs requries defining common process to ensure synchronization between SIGs and uniformity of the project.
 
-1. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository. [Design proposal template](https://github.com/kubevirt/community/blob/main/design-proposals/proposal-template.md)
-2. **SIG Review and Collaboration**: Each VEP will have a target SIG, and the SIG will assign a dedicated reviewer to oversee the proposal, collaborate with other SIGs as needed, and provide feedback or veto when necessary.
-3. **Centralized Prioritization**: At the start of each release cycle, all accepted VEPs will be designated as the project’s priority, focusing community efforts on the associated pull requests. Acceptance will be based on community support and a commitment to implementation.
-4. **Visibility and Tracking**: The Author of an accepted VEPs will open an issue to track their progress, maturity stages (alpha, beta, GA), list the associated bugs, and user feedback
-5. **Single source of truth**: Each VEP will be the authoritative reference for the associated feature. This aligns with the Kubernetes KEP process. It will ensure that each enhancement
-Includes all the relevant information, including the design and the state.
+**Glossary**
+- VEP
+- EF
+- FF
+- RC
+- Alpha/Beta
+- release team
 
-The VEP owner is responsible to update it as its development progresses, until it is fully mature (or deprecated).
+## Process (DRAFT 2nd iteration)
+
+1. **VEP Creation**: VEP authors will initiate proposals via PRs to the `kubevirt/enhancements` repository. See what is required for every VEP - [Design proposal template](https://github.com/kubevirt/community/blob/main/design-proposals/proposal-template.md), each section needs to be filled even if the section doesn't apply to the VEP. 
+
+2. **Visibility and Tracking**: The Author of VEP will open an issue to track their progress, maturity stages (alpha, beta, GA), list the associated bugs, and user feedback. 
+See what is required for every VEP's issue [Issue template](https://github.com/kubevirt/community/blob/main/design-proposals/proposal-template.md), each section should be filled.
+
+3. **SIG Review and Collaboration**: Each VEP will have a target SIG or more in case of cross SIG feature, and the SIGs will assign a dedicated reviewer to oversee the proposal, collaborate with others SIGs as needed, and provide feedback or veto when necessary. The author and reviewers work towards merged VEP.
```

Comment:
```suggestion
3. **SIG Review and Collaboration**: Each VEP will have one target SIG or more in case of a cross SIG feature, and the SIGs will assign a dedicated reviewer to oversee the proposal, collaborate with others SIGs as needed, and provide feedback or veto when necessary. The author and reviewers work towards merged VEP.
```